backend/src/Designer/TypedHttpClients/ResourceRegistryOptions/ResourceRegistryOptionsClient.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Altinn.Studio.Designer.Models;
+using PolicyAdmin.Models;
+
+namespace Altinn.Studio.Designer.TypedHttpClients.ResourceRegistryOptions
+{
+    public class ResourceRegistryOptionsClients : IResourceRegistryOptions
+    {
+        HttpClient _client;
+
+        public ResourceRegistryOptionsClients(HttpClient client)
+        {
+            _client = client;
+        }
+
+        public async Task<EuroVocTerms> GetEuroVocTerms()
+        {
+            string url = "https://data.norge.no/reference-data/eu/eurovocs";
+
+            EuroVocTerms eurovoc;
+
+            try
+            {
+                HttpResponseMessage response = await _client.GetAsync(url);
+                string content = await response.Content.ReadAsStringAsync();
+                eurovoc = System.Text.Json.JsonSerializer.Deserialize<EuroVocTerms>(content, new System.Text.Json.JsonSerializerOptions() { PropertyNamingPolicy = System.Text.Json.JsonNamingPolicy.CamelCase });
+                return eurovoc;
+            }
+            catch (Exception ex)
+            {
+                throw new Exception($"Something went wrong when retrieving eurovocs", ex);
+            }
+        }
+
+        public async Task<LosTerms> GetLosTerms()
+        {
+            string url = "https://data.norge.no/reference-data/los/themes-and-words";
+
+            LosTerms losTerms;
+
+            try
+            {
+                HttpResponseMessage response = await _client.GetAsync(url);
+                string content = await response.Content.ReadAsStringAsync();
+                losTerms = System.Text.Json.JsonSerializer.Deserialize<LosTerms>(content, new System.Text.Json.JsonSerializerOptions() { PropertyNamingPolicy = System.Text.Json.JsonNamingPolicy.CamelCase });
```

Comment:
The naming policy can be initialized once at the top and reused

Reply:
fixed